packages/styles/scss/components/data-table/_data-table.scss

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use 'data-table-action';
```

Comment:
Along with `@use` I think we'll need to `@forward` any variables for modules that teams could potentially configure.

Reply:
Yep, thanks for catching this. I found 5 variables but 4 of them have no usage when I find all - I'll make note of them in a review, I think we can delete them